interp/src/interpreter/steppers/component_interpreter.rs

```diff
@@ -59,6 +59,7 @@ pub struct ComponentInterpreter {
     go_port: RRC<Port>,
     input_hash_set: Rc<HashSet<*const ir::Port>>,
     qual_name: ComponentQIN,
+    full_name_clone: ir::Id, // only for printing
```

Comment:
Use a doc comment here and add a little more detail about what kind of printing this is referring to.